src/main/java/org/folio/verticles/AbstractConsumersVerticle.java

```diff
@@ -84,12 +94,18 @@ protected Future<KafkaConsumerWrapper<String, String>> createConsumer(T eventTyp
       .processRecordErrorHandler((t, r) -> log.error("Failed to process event: {}", r, t))
       .build();
 
-    log.info("createConsumer:: moduleId={}", MODULE_ID);
-    return consumerWrapper.start(handler, MODULE_ID)
+    String consumerName;
+    if (Objects.nonNull(methodName)) {
```

Comment:
Added to produce a unique consumer group to avoid work stealing on `inventory.holdings-record` by `CREATE` consumer, `mod-orders` has Data Import consumer which use this consumer name pattern